src/main/java/uk/gov/pay/ledger/transaction/search/model/TransactionView.java

```diff
@@ -331,6 +337,7 @@ public String getGatewayPayoutId() {
         private Source source;
         private String walletType;
         private String gatewayPayoutId;
+        private Payment sharedPaymentDetails;
```

Comment:
Can we TransactionView here similar to `parentTransaction` or a new one ? TransactionView is solely to serialising data and Payment/Refund domain objects are used as a layer in between dao and resource. Using Payment along with TransactionView could mix use cases